sympy/matrices/expressions/matexpr.py

```diff
@@ -340,8 +340,10 @@ def as_explicit(self):
         as_mutable: returns mutable Matrix type
 
         """
+        if not isinstance(self.rows, (SYMPY_INTS, Integer)) or not isinstance(self.cols, (SYMPY_INTS, Integer)):
+            raise ValueError('Matrix with symbolic shape cannot be represented explicitly')
```

Comment:
```suggestion
        if (not isinstance(self.rows, (SYMPY_INTS, Integer))
            or not isinstance(self.cols, (SYMPY_INTS, Integer))):
            raise ValueError('Matrix with symbolic shape ' \
                'cannot be represented explicitly')
```

Reply:
I can agree for the `if` line, but why break the string? Are there any maximum line length recommendations?  `.editorconfig` doesn't mention any

Reply:
In PEP8 recommendations, it encourages to use less than 80 characters per line.

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

And we recommend to follow the PEP8 convention